docs/samples/management-api-v2-walkthrough/7-transfer-process.md

```diff
@@ -0,0 +1,89 @@
+# Initiation a Transfer Process
+
+## Old plain JSON Schema
+
+```json
+{
+  "assetId": "asset-id",
+  "connectorAddress": "provider-dsp-url",
+  "contractId": "contract-agreement-id",
+  "dataDestination": {
+    "properties": {
+      "type": "HttpProxy"
+    }
+  },
+  "managedResources": false,
+  "privateProperties": {
+    "receiverHttpEndpoint": "backend-service-endpoint"
+  },
+  "protocol": "ids-protocol-http",
+  "transferType": {
+    "contentType": "application/octet-stream",
+    "isFinite": true
+  }
+}
+```
+
+## New JSON-LD Document
+
+> Please note: In our samples, properties **WILL NOT** be explicitly namespaced, and internal nodes **WILL NOT** be typed, relying on `@vocab` prefixing and root schema type inheritance respectively.
+
+```json
+{
+  "@context": {
+    "@vocab": "https://w3id.org/edc/v0.0.1/ns/",
+    "odrl": "http://www.w3.org/ns/odrl/2/"
+  },
+  "assetId": "asset-id",
+  "connectorAddress": "provider-dsp-url",
+  "contractId": "contract-agreement-id",
+  "dataDestination": {
+    "properties": {
```

Comment:
the `properties` field in a data address is implicit, `type` property can be put as a `dataDestination` property